GOVERNANCE.md

```diff
@@ -0,0 +1,67 @@
+# Prometheus Governance v1.0
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Prometheus community. Common terminology used in this governance document are listed below:
+
+* **Core Developers**: The stewards of the Prometheus community
+
+* **Committers**: Developers who have write access to a project
+
+* **Maintainers**: Committers who are leads on an individual project (MAINTAINERS.md)
+
+* **Projects**: Any repository in the Prometheus GitHub organization
+
+## Values
+
+The Prometheus developers and community are expected to follow the values defined in the CNCF [charter](https://www.cncf.io/about/charter/), including the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Furthermore, the Prometheus community strives for kindness, giving feedback effectively and building a welcoming environment.The Prometheus developer community also strives to operate by rough consensus for the majority of decisions and only resort to conflict resolution through a motion if consensus can’t be reached.
+
+## Core Developers
+
+As stewards of the project, Core Developers have additional duties beyond that of a maintainer. They are expected to represent the best interests of the project, above any employer or other company/organisational ties. The current Core Developers are:
+
+* Brian Brazil, Fabian Reinartz, Julius Volz
+
+New Core Developers are proposed by Core Developers on the prometheus-team@ mailing list, and voted on using the rules discussed in the voting section below. The current Core Developers are recognized as leaders of the Prometheus project as a whole, through their sustained and significant contributions across the project.
```

Comment:
I'd make the same arguments as around adding people to -team, though not quite as strong in this case (in practice the core devs would likely agree this in private first - part of the point of having a non-public -team list for me is to try and reduce that happening, so there's at least some visibility).